DfE.FindInformationAcademiesTrusts/Program.cs

```diff
@@ -258,6 +264,27 @@ private static void AddAuthenticationServices(WebApplicationBuilder builder)
             });
     }
 
+    private static void AddApiHttpClient(WebApplicationBuilder builder)
+    {
+        builder.Services.AddHttpClient("DfE.FindInformationAcademiesTrusts.API", httpClient =>
```

Comment:
Microsoft currently recommends using "Typed Clients" for registering new `HttpClient`s - https://learn.microsoft.com/en-us/dotnet/core/extensions/httpclient-factory#typed-clients